prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -68,7 +75,13 @@ class CustomFHIRFunctionsTests {
         // Individual function results are tested on their own unit tests.
         CustomFHIRFunctions.CustomFHIRFunctionNames.values().forEach {
             // todo: this is temporary until this code is moved
-            if (it != CustomFHIRFunctions.CustomFHIRFunctionNames.LivdTableLookup) {
+            if (it == CustomFHIRFunctions.CustomFHIRFunctionNames.ChangeTimezone) {
```

Comment:
Consider in the future generically checking how many parameters are needed and doing the test.

Reply:
I like the idea, might be tricky with the number and type of parameters mattering, but worth thinking about.